stable/ambassador/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 0.50.2
+appVersion: 0.50.3
 description: A Helm chart for Datawire Ambassador
 name: ambassador
-version: 1.1.4
+version: 1.1.5
 icon: https://www.getambassador.io/images/logo.png
 home: https://www.getambassador.io/
 sources:
```

Comment:
hmm, while you're in there can you add the other person in OWNERS to the list of maintainers?

Reply:
And change `Flynn` to `kflynn`. It should be your github userid.

Reply:
Updated, thanks!